Ability to create lexical graph only (#127)
neo4j/neo4j-graphrag-python: docs/source/user_guide_kg_builder.rst

        os.environ["OPENAI_API_KEY"] = "sk-..."

If OpenAI is not an option, see :ref:`embedders` to learn how to use other supported embedders.

The embeddings are added to each chunk metadata, and will be saved as a Chunk node property in the graph if
`create_lexical_graph` is enabled in the `EntityRelationExtractor` (keep reading).

Lexical Graph Builder
=====================

> Once the chunks are extracted and, if needed, embedded, a graph can be created. The lexical graph contains:

Reviewer: ```suggestion
Once the chunks are extracted and embedded (when required), a graph can be created. The lexical graph contains:
```

I think this reads better?
